tagger.py

```diff
@@ -35,13 +35,13 @@
 @click.option(
     "--source_path",
     help="The parent folder of the directory tree containing all music files you want to organize",
-    type=click.Path(exists=True, file_okay=False),
+    type=click.Path(exists=True, file_okay=False, resolve_path=True),
     required=True,
 )
 @click.option(
     "--destination_path",
     help="The parent folder where the music files are going to be organized",
-    type=click.Path(exists=True, file_okay=False),
+    type=click.Path(exists=True, file_okay=False, resolve_path=True),
```

Comment:
The destination path doesn't need to exist. It will be created recursively due to the intermediate organizing_pattern/organizing_tags derived directories, so the argument exists=True should be exists=False